state/protocol/inmem/encodable.go

```diff
@@ -86,10 +86,17 @@ func (snap EncodableSnapshot) LatestSealedResult() (*flow.ExecutionResult, error
 	return nil, fmt.Errorf("LatestSealedResult: unreachable for correctly formatted sealing segments")
 }
 
+// EncodableFullDKG encodes DKG data for all participants and the group key. Effectively encodes output of the DKG.
 type EncodableFullDKG struct {
```

Comment:
DKG stands for _distributed_ key generation. But we only use this the encode the output of a _centralized_ key generation! Furthermore, the struct contains the key set for a threshold signature scheme (which is conceptually unrelated to a DKG)  Tarak has repeatedly reminded us to not conflate the terminology of Threshold Key Cryptography (in our particular case used for the Random Beacon) with DKG. 

As we are touching this code, I would suggest to also clean up the terminology:
* rename `EncodableFullDKG` to `ThresholdKeySet`
* rename `EncodableDKGParticipant` to `ThresholdParticipant`

```suggestion
// ThresholdKeySet contains the key set for a threshold signature scheme. Typically, the ThresholdKeySet is used to
// encode the output of a trusted setup. In general, signature scheme is configured with a threshold parameter t,
// which is the number of malicious colluding nodes the signature scheme is safe against. To balance liveness and
// safety, the Flow protocol fixes threshold to t = floor((n-1)/2), for n the number of parties in the threshold
// cryptography scheme, specifically n = len(Participants).
// Without loss of generality, our threshold cryptography protocol with n parties identifies the individual
// participants by the indices {0, 1, …, n-1}. The slice Participants is ordered accordingly. 
type ThresholdKeySet struct {
```